src/Stripe.net/Services/TaxRates/TaxRateCreateOptions.cs

```diff
@@ -0,0 +1,54 @@
+namespace Stripe
+{
+    using System;
+    using System.Collections.Generic;
+    using Newtonsoft.Json;
+    using Stripe.Infrastructure;
+
+    public class TaxRateCreateOptions : BaseOptions
+    {
+        /// <summary>
+        /// Whether the tax rate is currently available for new subscriptions.
+        /// </summary>
+        [JsonProperty("active")]
+        public bool? Active { get; set; }
+
+        /// <summary>
+        /// An arbitrary string attached to the object. Often useful for displaying to users.
+        /// </summary>
+        [JsonProperty("description")]
+        public string Description { get; set; }
+
+        /// <summary>
+        /// The name of the tax rate. This will not be shown to users and could be useful for you
+        /// to identify it internally.
```

Comment:
And this should be: "The display name of the tax rate, which will be shown to users."